Extensions/shorten_posts.js

```diff
@@ -21,10 +21,40 @@ XKit.extensions.shorten_posts = new Object({
 			text: "When to shorten posts",
 			type: "separator"
 		},
-		only_text: {
```

Comment:
Do we want to remove the option for the user to remove long text posts? What if they want to shorten only photosets? Do we want to support that use case?